docs/docs/telemetry.mdx

```diff
@@ -0,0 +1,122 @@
+---
+id: telemetry
+sidebar_label: Rasa Telemetry
+title: Rasa Telemetry
+---
+
+Rasa uses telemetry to report anonymous usage information. This information is
+essential help improve Rasa Open Source for all users.
+For the team working on Rasa Open Source it is important to understand how
+the product is used, it allows us to properly prioritize our research efforts
+and feature development.
+
+You will be notified about the telemetry reporting when running Rasa Open Source
+for the first time.
+
+## How to opt-out
+
+You can always opt-out from telemetry reporting using
+```bash
+rasa telemetry disable
+```
+
+or by setting `RASA_TELEMETRY_ENABLED=false` as an environment variable.
+
+If you want to enable reporting again, you can use
+```bash
+rasa telemetry enable
+```
+
+## Why do we use telemetry reporting?
+
+**Anonymous** telemetry data allow us to prioritize our research efforts
+and feature development based on usage. We want to collect aggregated
+information on usage and reliability so that we can ensure a high-quality product.
+
+So how will we use the reported telemetry data? Here are some examples
+what we use the data for:
+
+- We will be able to know which languages, pipelines and policies are used.
+  This will enable us direct our research efforts towards projects
+  that have the most impact on text and dialogue handling.
+- We will be able to know data set sizes and general structure (e.g. the number
+  of intents). This allows us to better test our software on different types
+  of data sets and optimize the frameworks performance.
+- We will be able to get more detail on the types of errors you are running
+  into while building an assistant (e.g. initialization, training, etc.).
+  This will let us improve the quality of our framework and better focus our
+  time on solving more common, frustrating issues.
+
```

Comment:
I'd suggest moving the sensitive data section up higher, after the ## Why do we use telemetry reporting? section. 

Your first thought after you read the examples of data usage is whether Rasa can read your training data and user messages. Maybe even renaming the section ## What do we *not* report?